repo/content/index_blob_manager.go

```diff
@@ -362,10 +341,10 @@ func (m *indexBlobManagerImpl) findCompactionLogBlobsToDelayCleanup(ctx context.
 }
 
 func (m *indexBlobManagerImpl) findBlobsToDelete(entries map[blob.ID]*cleanupEntry) (compactionLogs, cleanupBlobs []blob.ID) {
-	for _, e := range entries {
+	for k, e := range entries {
 		if e.age > m.maxEventualConsistencySettleTime {
 			compactionLogs = append(compactionLogs, e.BlobID)
-			cleanupBlobs = append(cleanupBlobs, e.BlobID)
+			cleanupBlobs = append(cleanupBlobs, k)
```

Comment:
IIUC: this was one of the issues, using the incorrect blob id for the cleanup blobs, is that correct?

Reply:
that's correct.